python/whylogs/api/store/query.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import Optional
+
+
+@dataclass
+class DateQuery:
+    start_date: datetime
+    end_date: Optional[datetime] = field(default=None)
+
```

Comment:
does no end_date mean implicitly end_date=now, or does it turn into a single date picker?

Reply:
it goes on the post_init method. if no end_date, `end_date == start_date`
also note there that if `end_date < start_date`, then i switch them